tuxemon/core/states/combat/combat_animations.py

```diff
@@ -432,12 +432,12 @@ def animate_update_party_hud(self, player, home):
 
         for index in range(player.party_limit):
             sprite = None
-            if len(player.monsters) - count_fainted > index: # render alive Tuxemon balls
+            if len(player.monsters) - count_fainted > index:  # render alive Tuxemon balls
```

Comment:
I have autopep installed in PyCharm and it made these changes.

Reply:
thats fine.  minor linting is not a problem